src/Platforms/PostgreSQL94Platform.php

```diff
@@ -755,6 +757,28 @@ public function getCreateSchemaSQL($schemaName)
         return 'CREATE SCHEMA ' . $schemaName;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getDropIndexSQL($index, $table = null)
+    {
+        if ($index instanceof Index && $index->isPrimary()) {
+            if (! ($table instanceof Table || ($table !== null && $table !== ''))) {
+                throw new InvalidArgumentException(
+                    __METHOD__ . '() expects $table parameter to be nonempty string or ' . Table::class . '.'
```

Comment:
The check above does not match this error message. For instance, you could pass an integer or an array here.

Reply:
Thank you for the hint. An explicit string type check also makes psalm happy.

Reply:
Let's squash the commits together and pretend this never happened then :stuck_out_tongue: ?

How to do that?

1. `git rebase -i origin/3.1.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already push. Don't forget the `--force` option otherwise git will try to merge both things together.